data/language/pt-BR.yml

```
@@ -2183,3 +2183,81 @@ strings:
+  2132: "{COLOUR WINDOW_2}Fator de escala da janela:"
+  2133: "{COLOUR BLACK}+"
+  2134: "{COLOUR BLACK}-"
+  2135: "{COLOUR BLACK}{INT32_1DP}"
+  2136: "{COLOUR WINDOW_2}Zoom para posição do cursor"
+  2137: "{SMALLFONT}{COLOUR BLACK}Quando ativado, o zoom centralizará no cursor ao invés do centro da tela"
+  2138: "{COLOUR WINDOW_2}Tocar música do menu"
+  2139: "Sair para o menu"
+  2140: "Sair do OpenLoco"
+  2141: "{COLOUR WINDOW_2}Desabilitar companhias de IA"
+  2142: "{SMALLFONT}{COLOUR BLACK}Isso desabilita o 'pensamento' da IA, as tornando inefetivas.{NEWLINE}Em novos jogos, isso também previne que novas companhias de IA se formem."
+  2143: "{COLOUR WINDOW_2}Quantidade de autosaves:"
+  2144: "{COLOUR WINDOW_2}Frequência de autosaves:"
+  2145: "Nunca"
+  2146: "Todo mês"
+  2147: "Cada {INT32} meses"
+  2148: "{COLOUR WINDOW_2}Gerador:"
+  2149: Original
+  2150: Melhorado
+  2151: "Modificar veículo"
+  2152: "Clonar veículo"
+  2153: "Impossível clonar veículo..."
+  2154: "Localizar veículo na visão principal"
+  2155: "Seguir veículo na visão principal"
+  2156: "Habilitar menu de trapaças/debugging"
+  2157: "{SMALLFONT}{COLOUR BLACK}Isso adiciona um botão extra na barra de tarefas do jogo, permetindo fácil acesso a certas trapaças e opções de debug."
+  2158: "Habilitar modo sandbox"
+  2159: "Permitir direção manual"
+  2160: "Permitir construir enquanto pausado"
+  2161: "Mostrar contatdor de FPS"
+  2162: "{SMALLFONT}{COLOUR BLACK}Isso mostrará um contato no topo da tela, indicando o númer de frames desenhados por segundo."
+  2163: "Tirar limite de FPS"
+  2164: "{SMALLFONT}{COLOUR BLACK}Remove a restrição de renderização em 40Hz."
+  2165: "Hardware"
+  2166: "Renderização de mapa"
+  2167: "Inspetor de Tiles"
+  2168: "{SMALLFONT}{COLOUR BLACK}Ative para selecionar um tile no para para inspecionar."
+  2169: "Superfíce"
+  2170: "Via"
+  2171: "Estação"
+  2172: "Semáforo"
+  2173: "Construção"
+  2174: "Árvore"
+  2175: "Parede"
+  2176: "Estrada"
+  2177: "Indústria"
+  2178: "{COLOUR WINDOW_2} {STRINGID} ({STRINGID})"
+  2179: "{COLOUR WINDOW_2} {STRINGID} - {STRINGID} ({STRINGID})"
+  2180: "X:"
+  2181: "Y:"
+  2182: "{COLOUR WINDOW_2}{INT16}"
+  2183: "Dados dos elementos do tile"
+  2184: "Trapaças"
+  2185: "Trapças financeiras"
+  2186: "Trapaças de companhia"
+  2187: "Trapaças de veículos"
+  2188: "Trapaças de municípios"
+  2189: "Limpar empréstimo"
+  2190: "Limpar"
+  2191: "{COLOUR BLACK}{CURRENCY32}"
+  2192: "Selecionar companhia alvo"
+  2193: "Selecionar trapaça para aplicar"
+  2194: "Mudar controle para esse companhia"
+  2195: "Adiquirir todas posses de companhia"
+  2196: "Ativar falência"
+  2197: "Ativar estado de cadeia"
+  2198: "Aumentar fundos"
+  2199: "{COLOUR WINDOW_2}Quantidade:"
+  2200: "Adicionar"
+  2201: "Confiabilidade de veículo"
+  2202: "Mudar todas para 0 (ilimitada)"
+  2203: "Mudar todas para 100 (renovação completa)"
+  2204: "mudar avaliação da companhia"
```

Any particular reason this does not start with a capital? I suggest changing to the following:
```suggestion
  2204: "Mudar avaliação da companhia"
```

---

Sure! It may have passed by, thanks!